tests/routes/crate/version/docs-link-test.js

```diff
@@ -81,4 +81,19 @@ module('Route | crate.version | docs link', function (hooks) {
     await visit('/crates/foo');
     assert.dom('[data-test-docs-link] a').hasAttribute('href', 'https://docs.rs/foo/0.6.2');
   });
+
+  test('null/undefined builds in docs.rs responses are ignored', async function (assert) {
+    this.server.create('crate', { name: 'foo', documentation: 'https://docs.rs/foo/0.6.2' });
+    this.server.create('version', { crateId: 'foo', num: '0.6.2' });
+
+    this.server.get('https://docs.rs/crate/:crate/:version/builds.json', [undefined]);
```

Comment:
`undefined` is not a valid JSON value, so this should be impossible from what I can tell. `docsBuilds[0]` will return `undefined` though if `docsBuilds == []`.

```suggestion
    this.server.get('https://docs.rs/crate/:crate/:version/builds.json', []);
```